src/main/java/com/xebia/incubator/xebium/SeleniumDriverFixture.java

```diff
@@ -456,6 +524,81 @@ private boolean executeDoCommand(final String methodName, final String[] values)
 		}
 	}
 
+	private String encryptString(String password) { //encrypt exposed passwords or information stub
```

Comment:
I think the encryption/decryption methods should move to their own fixture, outside of Xebium.